articles.go

```diff
@@ -44,5 +45,61 @@ func articlesLogic() {
 			log.Fatal(err)
 			os.Exit(1)
 		}
+
+		// ID of the created article
+		id := response["id"]
+
+		// Get Data from article to make petitions accordingly
+		metaInfoItems := []string{"tags", "references"}
+
+		for _, meta := range metaInfoItems {
+			metaInfo := data["data"].(map[string]interface{})["attributes"].(map[string]interface{})["meta"].(map[string]interface{})[meta].([]interface{})
```

Comment:
this line is too long 😮

Reply:
Wooooo!

Reply:
hahah, yeah is nasty!

XD

Reply:
Does anyone of you know a better approach for split this kind of line? :-P

Reply:
you can do something like:
~~~golang
article := data["data"].(map[string]interface{})
attributes := article["attributes"].(map[string]interface{})
metaInfo := attributes["meta"].(map[string]interface{})[meta].([]interface{})
~~~

maybe have more lines but improves readability

Reply:
That is not just more lines (which is irrelevant, really); this has more allocation in memory, maybe not a big impact, but a more allocation of memory centainly.

Improves readability (which is good). I'm going to do it that way